xtask/src/qemu.rs

```diff
@@ -313,10 +466,20 @@ pub fn run_qemu(arch: UefiArch, opt: &QemuOpt) -> Result<()> {
         }
     }
 
+    let tmp_dir = TempDir::new()?;
+    let tmp_dir = tmp_dir.path();
+
     // Set up OVMF.
     let ovmf_paths = OvmfPaths::find(opt, arch)?;
-    add_pflash_args(&mut cmd, &ovmf_paths.code, /*read_only=*/ true);
-    add_pflash_args(&mut cmd, &ovmf_paths.vars, ovmf_paths.vars_read_only);
+
+    // Make a copy of the OVMF vars file so that it can be used
+    // read+write without modifying the original. Under AArch64, some
+    // versions of OVMF won't boot if the vars file isn't writeable.
+    let ovmf_vars = tmp_dir.join("ovmf_vars");
+    fs_err::copy(&ovmf_paths.vars, &ovmf_vars)?;
```

Comment:
Hmm, but isn't the whole point of the `OVMF_VARS.fd` file to be a persistent store of information for the firmware? If we always just copy it to a temporary location, this will make it start from scratch each time.

Reply:
That's true, but I don't think that really matters for `uefi-test-runner` since we don't set or check any persistent values.